bowtie/_cli.py

```diff
@@ -1198,6 +1203,14 @@ def convert(
 @SET_SCHEMA
 @TIMEOUT
 @VALIDATE
+@click.option(
```

Comment:
If this isn't meant to be exposed, better to have it be an environment variable we check probably, which is a bit more common.

Reply:
I am not sure how that would happen, then we would have to somehow set the enivornment variable from inside bowtie itself before running the pyperf benchmark. But idts the env var would persist when pyperf then runs its own process and tries running `bowtie run`. So when bowtie run would then check whether the env var exists or not, it would always say it doesnt exist, no ? @Julian

Reply:
I tried doing this, the env variable didnt persist. Any other idea for the same?

Reply:
Yeahh @Julian , ig when pyperf creates a process of its own, it doesnt inherit the env variables as also mentioned in : https://github.com/psf/pyperf/issues/47

Reply:
Oh wait, they did introduce some flag later on to enable inherting them!

Reply:
Works now @Julian.